MSET9_installer_script/mset9.py

```diff
@@ -148,6 +167,24 @@ def remove():
 	mode=0
 	print(" done.")
 
+def softcheck(keyfile, size, crc32, retval):
+	if not os.path.exists(keyfile):
+		print("%s \ndoes not exist on SD card!" % keyfile)
+		return retval
+	elif size:
+		s=os.path.getsize(keyfile)
```

Comment:
please, use a linter on this or at least put a freaking space before and after the equals sign like a normal human

Reply:
correction: average zoogie code formatting